CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## [1.11.3](https://github.com/brave/brave-ios/releases/tag/v1.11.3)
+
+ - Added support for U2F/WebAutn hardware security keys. ([#1406](https://github.com/brave/brave-ios/issues/1406))
+ - Fixed app crash when switching from Private mode to normal mode. ([#1309](https://github.com/brave/brave-ios/issues/1309))
```

Comment:
nit: I don't think we need to have `app` in there. Maybe reword to something like:

```
Fixed crash when switching from private browsing to regular browsing mode.
```

Reply:
Helps differentiate between app crash vs webview crash. The same convention is used on Android as well.

Reply:
Normal tabs/mode is the generally used convention on all platforms and specifically on iOS. Should probably use the same here as well.